ui/src/pages/_app.tsx

```diff
@@ -15,46 +15,29 @@ import '../styles/globals.css';
 
 type CustomProps = {
     protectedPage?: boolean;
-    metadata?: {
-        title?: string;
-        description?: string;
-    };
 };
 
 const App = ({ Component, pageProps }: Types.AppProps<CustomProps>) => {
-    const [mounted, setMounted] = useState(false);
     const { user } = Stores.useAuthStore();
     const { darkMode } = Stores.usePreferencesStore();
 
-    const metadata = {
-        title: pageProps.metadata?.title ? pageProps.metadata.title : 'Octopus',
-        description: pageProps.metadata?.description
-            ? pageProps.metadata.description
-            : 'Free, fast and fair: the global primary research record where researchers publish their work in full detail.'
-    };
-
     // check authentication client side
     Hooks.useAuthCheck(pageProps.protectedPage || false);
 
     // matomo tracking
     Hooks.useMatomoNext();
 
     useEffect(() => {
-        setMounted(true);
-
-        return () => {
-            setMounted(false);
-        };
+        Stores.usePreferencesStore.persist.rehydrate();
+        Stores.useAuthStore.persist.rehydrate();
     }, []);
 
     return process.env.NEXT_PUBLIC_MAINTENANCE_MODE === 'true' ? (
-        mounted && <Components.Maintenance />
+        <Components.Maintenance />
     ) : (
         <Contexts.ConfirmationModalProvider>
             <Head>
                 <meta name="viewport" content="width=device-width, initial-scale=1" />
-                <meta name="og:description" content={metadata.description} />
-                <meta name="og:title" content={metadata.title} />
```

Comment:
I don't think there's any real benefit if we do that tbh, at least not in our case. Why would we introduce duplicate meta tags and then dedupe them (if we don't need to)? That could lead to other issues. There may be some specific use cases for that, but for our project, i can't find any atm.

Reply:
The twitter tags item originally asked for a default option for all non-publication pages: https://jiscdev.atlassian.net/browse/OC-95

A lot of the pages now have their own values since I added them during that item, but it's still worth having a fallback/default option to provide coverage when we have (or introduce) a page without its own meta tag values.

Reply:
I see. I added the fallback meta tags back since they are harmless.

Reply:
Nice, I must have been wrong about using "key" as well as it pulls through the right values without those.